src/chrome/content/rules/GEBlogs.com.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+	Invalid certificate:
+		*.files.geblogs.com
+		files.*.geblogs.com
+
+	Other subdomains are mismatched or refused.
```

Comment:
@Bisaloo could you add some more documentation about the behavior of this host?
E.g. why a wildcard is required, but only specific patterns are rewritten.

Reply:
How would you describe this? I don't know how to explain it better than what I did.